esp/src/src-react/components/forms/ZAPDialog.tsx

```diff
@@ -192,14 +192,14 @@ export const ZAPDialog: React.FunctionComponent<ZAPDialogProps> = ({
                     if (key === "AbsoluteTimeRange") {
                         const startDate = logFilter.AbsoluteTimeRange.StartDate ? new Date(logFilter.AbsoluteTimeRange.StartDate).toISOString() : "";
                         const endDate = logFilter.AbsoluteTimeRange.EndDate ? new Date(logFilter.AbsoluteTimeRange.EndDate).toISOString() : "";
-                        formData.append("LogFilter.AbsoluteTimeRange.StartDate", startDate);
-                        formData.append("LogFilter.AbsoluteTimeRange.EndDate", endDate);
+                        formData.append("LogFilter_AbsoluteTimeRange_StartDate", startDate);
+                        formData.append("LogFilter_AbsoluteTimeRange_EndDate", endDate);
                     } else {
-                        formData.append(`LogFilter.${key}`, logFilter[key]);
+                        formData.append(`LogFilter_${key.replace(/\./, "_")}`, logFilter[key]);
```

Comment:
@jeclrsg should this be an url encode?

Reply:
It was unnecessary; I thought there were some fields with "." in the name, but there were not.